ERRORCODES.md

```diff
@@ -82,6 +82,7 @@ The following warnings are currently emitted by default:
 | <a id="Y066" href="#Y066">Y066</a> | When using if/else with `sys.version_info`, put the code for new Python versions first. | Style
 | <a id="Y067" href="#Y067">Y067</a> | Don't use `Incomplete | None = None` in
 argument annotations. Instead, just use `=None`. | Style
+| <a id="Y068" href="#Y068">Y068</a> | Don't use `@override` in stub files. | Correctness
```

Comment:
Could we maybe give an explanation here, such as "Other tools such as stubtest are better placed to catch issues where stubs deviate from the runtime code, so `@override` just adds visual noise to the stub".

I'd also possibly classify this one under the "Understanding stubs" category? It seems like it's a tool that only really makes sense for runtime code, so it arguably demonstrates a misunderstanding of how stubs work if you use it in a stub